WORKING_GROUPS.md

```diff
@@ -24,12 +24,17 @@ Working Group membership or normal TSC motion and vote.
 
 ## Current Working Groups
 
-NOTE: Technical Working Groups exist in the [nodejs/node repository](https://github.com/nodejs/node/blob/master/WORKING_GROUPS.md).
+NOTE: Technical Working Groups exist in the [nodejs/ctc repository](https://github.com/nodejs/ctc/blob/master/WORKING_GROUPS.md).
+
+## De-chartered Working Groups
 
 * [Inclusivity](#inclusivity)
 
 ### [Inclusivity](https://github.com/nodejs/inclusivity)
 
+*Note*: The efforts and mission of the Inclusivity Working Group has been
```

Comment:
Grammar correction: has -> have   (It's "the efforts and mission" that have been picked up.)